playwright-e2e/tests/dsm/osteo/os1-to-os2-display-verification.spec.ts

```diff
@@ -623,18 +622,35 @@ test.describe.serial(`${StudyName.OSTEO} -> ${StudyName.OSTEO2}: Verify expected
     await researchConsentActivityForOS1.scrollIntoViewIfNeeded();
     await expect(researchConsentActivityForOS1).toBeVisible();
 
-    const researchConsentActivityForOS2 = await surveyDataTab.getActivity({
+    /* OS2 has had 2 different versions of consent: version 2 and version 3 - occasionally, a re-consented participant with the former is selected by the test */
+    const clinicalConsentVersion2 = await surveyDataTab.getActivity({
       activityName: SurveyName.RESEARCH_CONSENT_FORM,
-      activityVersion: ActivityVersion.THREE
+      activityVersion: ActivityVersion.TWO,
+      checkForVisibility: false
     });
+    const clinicalConsentVersion3 = await surveyDataTab.getActivity({
+      activityName: SurveyName.RESEARCH_CONSENT_FORM,
+      activityVersion: ActivityVersion.THREE,
+      checkForVisibility: false
+    });
+    const researchConsentActivityForOS2 = (await (clinicalConsentVersion2).isVisible()) ? clinicalConsentVersion2 : clinicalConsentVersion3;
```

Comment:
seemed to be the simplest way to account for times when the re-consented participant the test picks has an earlier version of an OS2 activity